code/modules/clothing/masks/miscellaneous.dm

```diff
@@ -16,6 +16,16 @@
 			return
 	..()
 
+/obj/item/clothing/mask/muzzle/breath // Must be under /mask/muzzle/ because the speech obstruction is tied to /mask/muzzle rather than being a cloting flag. *sigh
```

Comment:
:-1:

Reply:
Make it an element.

Reply:
Could I add it as a clothing flag, and then alter the /mob/living/carbon/is_muzzled() function to check for this clothing flag?